src/utils/CopyToClipboardBtn/index.tsx

```diff
@@ -54,7 +54,7 @@ const CopyToClipboardBtn = ({
   };
 
   const onButtonBlur = (): number =>
-    setTimeout((): void => setClicked(false), 300);
+    window.setTimeout((): void => setClicked(false), 300);
```

Comment:
why was this needed? if someone would use it in SSR env it will fail. If we need to use the window it's better to check with `if (typeof window !== 'undefined)`

Reply:
There is a typescript error in that line when calling setTimeout directly we can leave as it was until now

Reply:
Now, it seems we have to take some action there
https://github.com/gnosis/safe-react-components/pull/146/checks?check_run_id=3359298170#step:4:70

Reply:
you can also not return it by wrapping the function body with curly braces and not including return statement
```
const onButtonBlur = (): void => {
  setTimeout((): void => setClicked(false), 300)
  }